staging/src/k8s.io/component-base/logs/options.go

```diff
@@ -70,11 +70,23 @@ func (o *Options) Validate() []error {
 	return errs
 }
 
-func flagIsSet(name string) bool {
+// UnNormalize replaces hyphens with underscores
+// we should always use underscores instead of hyphens when validate flags
+func UnNormalize(s string) string {
+	return strings.Replace(s, "-", "_", -1)
+}
```

Comment:
since this is only used locally, don't export it

I'd also suggest a more descriptive name like `hyphensToUnderscores`